kaspresso/src/main/kotlin/com/kaspersky/kaspresso/kaspresso/Kaspresso.kt

```diff
@@ -863,6 +864,7 @@ data class Kaspresso(
             if (!::viewBehaviorInterceptors.isInitialized) viewBehaviorInterceptors =
                 if (isAndroidRuntime) mutableListOf(
                     AutoScrollViewBehaviorInterceptor(autoScrollParams, libLogger),
+                    AutoScrollViewFallbackInterceptor(autoScrollParams, libLogger),
```

Comment:
We've wanted to make this Interceptor optional.
That's why it's better to remove new interceptor here.